### DISCUSSION TITLE
Running in Linux, tab-completions need an extra Tab

### DISCUSSION THREAD

**Discussion by wsande** [python-cmd2/cmd2#1166] [Q&A]
I tried running my CLI on a Linux server where the system for which it's providing an interface is running. It mostly works fine, except that I need an extra TAB to do tab-completion.   

The system is running Centos 8, and I copied my python script there and pip-installed cmd2.  I'm logged into that server via an ssh session, using SecureCRT on my Windows machine.

When I hit TAB the first time, I hear the default error sound on my Windows machine, and when I hit TAB the second time, it tab-completes correctly.  

Ayn idea why this is happening and how to fix it?  Running the python script on my Windows machine it works perfectly.

**Comment by tleonhardt** [comment]
This is not an issue with `cmd2` - the vast majority of our users are running on Linux and it works as expected.  It must be a problem in your setup.  I'd recommend looking at preferences for SecureCRT to see if there is something to configure?
